pkg/cluster/deploybaseresources.go

```diff
@@ -35,6 +38,47 @@ func (m *manager) createDNS(ctx context.Context) error {
 	return m.dns.Create(ctx, m.doc.OpenShiftCluster)
 }
 
+func (m *manager) createOIDC(ctx context.Context) error {
+	if m.doc.OpenShiftCluster.Properties.ServicePrincipalProfile != nil || m.doc.OpenShiftCluster.Properties.PlatformWorkloadIdentityProfile == nil {
+		return nil
+	}
+
+	blobContainerName := env.OIDCBlobContainerPrefix + m.doc.ID
+
+	publicAccess := azstorage.PublicAccessNone
+	// Public access on OIDC Container needed for development environments because of no AFD avaialbility
+	if m.env.IsLocalDevelopmentMode() {
+		publicAccess = azstorage.PublicAccessBlob
+	}
```

Comment:
This would be better as a feature flag on the RP rather than an `IsLocalDevelopmentMode`.  

It would allow us to toggle the functionality in INT if needed or if we needed to decouple from AFD in the future.

Reply:
Resolving this thread since the change has been made and the motivation for the change is clear.